docs/user/tutorials/new.rst

```diff
@@ -105,6 +105,12 @@ build the docs using the commands above.
 Once the docs build has passed, you can use the Settings on the repo page on the
 GitHub website to enable github pages publishing of the ``gh-pages`` branch.
 
+Setting up pre-commit
+---------------------
+
+To install the pre-commit see `../../developer/how-to/lint`.
+
+
```

Comment:
I feel this section should be noted in and/or come before the "running the tests" section, as that's what this pre-commit does. 

I think it should certainly go before the "Pushing to GitHub" section, as by that point the user has already clearly done a commit and so missed the initial opportunity to run the pre-commit.

Reply:
Putting it in its own section before "running the tests"